licenses.d2iq.yaml

```diff
@@ -470,7 +470,7 @@ resources:
         notice_path: NOTICE
         ref: ${image_tag}
         url: https://github.com/prometheus/node_exporter
-  - container_image: quay.io/prometheus/prometheus:v2.55.1
+  - container_image: quay.io/prometheus/prometheus:v3.2.0
```

Comment:
let's 🤞 no bad breaking changes

Reply:
Yes I wanted to test sth with prom v3 and that's kinda main reason why I am doing this! Fwiw this is tested upstream so at least kubecost should be fine. I will keep an eye out for other impact if any.